w3d_translator/w3d_translator.py

```diff
@@ -52,17 +47,17 @@ def farewell_error():
 
 
 # Translate a single project
-def translate_project(project_dir: Path, out_dir: Path):
+def translate_project(project_dir: Path, out_dir: Path, dev: bool):
```

Comment:
maybe make `dev` a kwarg with a default of `false` - that way if this is used not via typer, no need to make that arg required

Reply:
nah, both cli and kwarg